### PR TITLE
Add Travis and README badge

### PR DESCRIPTION
**Pull request by vicens-aniol**
https://travis-ci.org/vicens-aniol/contact-groups-server `<----log`

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+language: node_js
+
+sudo: enabled
+
+script:
+  - echo "Hi, this is Travis!";
+  - echo "Travis CI";
+  - echo "I'm in JBoss Community"
```
Comment (m-murad) [inline .travis.yml]: Remove these three lines.

File: .travis.yml [hunk 2]
```diff
@@ -1,9 +1,8 @@
 language: node_js
 
-sudo: enabled
+sudo: mongodb
 
 script:
   - echo "Hi, this is Travis!";
```
Comment (m-murad) [inline .travis.yml]: Remove this line

**Comment by vicens-aniol** [comment]
Updated @m-murad

**Comment by shubhamvashisht** [comment]
@vicens-aniol squash the commits!

**Comment by vicens-aniol** [comment]
@shubhamvashisht Squashed

**Pull request merged**